utils/logalloc.hh

```diff
@@ -209,6 +209,12 @@ class region_group {
     };
 
     struct on_request_expiry {
+        class blocked_requests_timed_out_error : public timed_out_error {
+        public:
+            virtual const char* what() const noexcept override {
+                return "region_group::_blocked_requests timed out";
```

Comment:
For the same reasons as in the commitlog we would want to print `memtable: timed out`. That requires the ability to assign a name to a `region_group`.

Currently all region_groups are created on behalf of memtables (in dirty_memory_manager), but logalloc should not know about this as we want to keep it abstract and move to seastar one day.

Reply:
Thanks, done.